man/meson.build

```diff
@@ -115,17 +130,23 @@ systemd_directives_xml = custom_target(
         input : ['directives-template.xml', source_xml_files],
         output : 'systemd.directives.xml',
         depends : custom_entities_ent,
-        command : [make_directive_index_py, '@OUTPUT@', '@INPUT@'])
+        command : [make_directive_index_py, '@OUTPUT@', '@INPUT@'],
+)
 
 nonindex_xml_files = source_xml_files + [systemd_directives_xml]
 systemd_index_xml = custom_target(
         'systemd.index.xml',
         input : nonindex_xml_files,
         output : 'systemd.index.xml',
-        command : [make_man_index_py, '@OUTPUT@'] + nonindex_xml_files)
-
-foreach tuple : xsltproc.found() ? [['systemd.directives', '7', systemd_directives_xml],
-                                    ['systemd.index',      '7', systemd_index_xml]] : []
+        command : [make_man_index_py, '@OUTPUT@'] + nonindex_xml_files,
+)
+
+foreach tuple : (
+        xsltproc.found() ? [
+                ['systemd.directives', '7', systemd_directives_xml],
+                ['systemd.index', '7', systemd_index_xml],
+        ] : []
+)
```

Comment:
No.

Reply:
This should be reworked to use dicts anyway (and have it not defined inline). This is a code smell to me.